system/Router/RouteCollection.php

```diff
@@ -1699,7 +1700,9 @@ public function resetRoutes()
      */
     protected function loadRoutesOptions(?string $verb = null): array
     {
-        $verb = $verb ?: $this->getHTTPVerb();
+        if (null === $verb || $verb === '') {
```

Comment:
Do we need `$verb === ''` check?

Reply:
because `$verb` is a `string` we also need to validade if its empty string to replace the validation `$verb = $verb ?: $this->getHTTPVerb();` unless we check like this:

```
if (! $verb) {
    $verb = $this->getHTTPVerb();
}
```

what do you sugest?

Reply:
Then, what if `'0'`?
Probably we don't expect `'0'` for `$verb`.

I'm not sure what is correct. The question is just a question.
So if you believe you should check `''`, it is fine.

By the way, we do not use Yoda conditions.

Reply:
After another review of all uses of this method I think its reasonable to only validate the `null`value.

I will  fix the commit.